### DISCUSSION TITLE
Replacing multiple dats with a single parkobj

### DISCUSSION THREAD

**Discussion by spacek531** [OpenRCT2/OpenRCT2#16762] [Q&A]
In NEDesigns custom scenery history, there are numerous examples of custom scenery that were released with problems then later fixed with the fixed version being differentiated by adding "fixed" in the name. There are also numerous examples of scenery with identical graphics but different identifiers and possibly different rotation. I would like to conglomerate these otherwise-identical or versioned scenery into a single parkobj that replaces them while at the same time helping with some draw order issues they face: https://github.com/OpenRCT2/OpenRCT2/issues/15348

I know the originalId system exists and works for single dats, but I don't know if the same is true for multiple dats. Is there such a thing in the parkobj circle? I know there are tables hardcoded in the game, but that is not feasible for custom scenery.

I see the hardcoded list as feasible for OpenRCT2 scenery, so my next question is, is OpenRCT2 willing to expand its collection of 'official' objects?

**Comment by 733737** [comment]
if there is a specific use for cso other than aesthetics, that is your best candidate for official objects. Pragmatism is more important, since lots of cso is modified from game assets which is questionable, and every cso bundled with game increase the footprint size of the install and increases compile time
